### DISCUSSION TITLE
아까 논의했던게 또 이상하네요...

### DISCUSSION THREAD

**Discussion by ppskj178** [boostcampaitech2/object-detection-level2-cv-06#95] [ideas]
내일 빠른 모델테스트를 위해 스케줄러 테스트중인데....

스케줄러에따라서 지표가 천차만별이네요.... 자세한건 wandb 참고 바라고
의견 부탁드립니다... 알수가없네요...
![image](https://user-images.githubusercontent.com/20790778/136193231-1d2488eb-59e3-44e6-b443-efe328385acb.png)

**Comment by jaehwan-AI** [comment]
그래프랑 wandb를 봐서는 어떤 문제가 있는 지 감이 안오는데 혹시 추가 설명 부탁드려도 될까요?

**Reply by ppskj178** [reply]
아까 해상도와 mAP_m, s 관계에서 해상도 높히니까  m 값이 바닥을 기는게 이상한 데이터들이 다 s로 가지않았나 라고 해석했는데

lr을 높게줬더니 m값이 살아났어요....

**Reply by ppskj178** [reply]
새로운 추측은 저번 결과는 lr이 너무 낮아서 로걸옵티마에 갖혔던게 아닐까 싶어요

**Reply by jaehwan-AI** [reply]
확실히 wandb 그래프를 보니까 그런 것 같네요
그래도 m과 s가 절대적으로 수치상 낮은 것은 확실한 것 같아요

**Reply by JiyouSeo** [reply]
지금 wandb 확인하고 있는데 valid_cascade_1024_1x_schedule_test가 mAP_m부분은 해상도 조절 없이 train한 mAP만큼 나와주고, mAP_l 부분은 가장 높게 나오네요. learning rate만 조절했다고 이런 결과가 나오다니..
